IMAGES.rst

```diff
@@ -0,0 +1,430 @@
+    --build-arg ENTRYPOINT_FILE="https://raw.githubusercontent.com/apache/airflow/1.10.10/entrypoint.sh" \
+    --build-arg SOURCES_FROM="Dockerfile" \
+    --build-arg SOURCES_TO="/Dockerfile" \
+    --build-arg WWW_FOLDER="www_rbac"
+
+Image manifests
+---------------
+
+Together with the main CI images we also build and push image manifests. Those manifests are very small images
+that contain only results of the docker inspect for the image. This is in order to be able to
+determine very quickly if the image in the docker registry has changed a lot since the last time.
+Unfortunately docker registry (specifically dockerhub registry) has no anonymous way of querying image
+details via API, you need to download the image to inspect it. We overcame it in the way that
+always when we build the image we build a very small image manifest and push it to registry together
+with the main CI image. The tag for the manifest image is the same as for the image it refers
+to with added ``-manifest`` suffix. The manifest image for ``apache/airflow:master-python3.6-ci`` is named
+``apache/airflow:master-python3.6-ci-manifest``.
+
+
+Pulling the Latest Images
+-------------------------
+
+Sometimes the image needs to be rebuilt from scratch. This is required, for example,
+when there is a security update of the Python version that all the images are based on and new version
+of the image is pushed to the repository. In this case it is usually faster to pull the latest
+images rather than rebuild them from scratch.
+
+You can do it via the ``--force-pull-images`` flag to force pulling the latest images from the Docker Hub.
+
+For production image:
+
+.. code-block::
+
+  ./breeze build-image --force-pull-images --production-image
+
+For CI image Breeze automatically uses force pulling in case it determines that your image is very outdated,
+however uou can also force it with the same flag.
+
+.. code-block::
+
+  ./breeze build-image --force-pull-images
+
+Using the images
+================
+
+Both images have entrypoint set as dumb-init with entrypoint.sh script executed (in order to forward
+signals). This entrypoint works as follows:
+
+* If ``AIRFLOW__CORE__SQL_ALCHEMY_CONN`` variable is passed to the container and it is either mysql or postgres
+  SQL alchemy connection, then the connection is checked and the script waits until the database is reachable.
+* If no ``AIRFLOW__CORE__SQL_ALCHEMY_CONN`` variable is set or if it is set to sqlite SQL alchemy connection
+  then db reset is executed.
+* If ``AIRFLOW__CELERY__BROKER_URL`` variable is passed and scheduler, worker of flower command is used then
+  the connection is checked and the script waits until the Celery broker database is reachable.
+* If ``DB_RESET`` is specified - the 'airflow db reset' is used.
+
+* If no argument is specified - you are dropped in bash shell.
+* If there are any arguments they are passed to "airflow" command
```

Comment:
A thought: should we add this or shorter info to docs? In this way, information about docker image will appear on Airflow website.

Reply:
I think more info and docs will come when we have helm chart. I see the Dockerfile as a starting point and it's the helm chart where we should see more use cases explained.